src/internal/config/config.go.tmpl

```diff
@@ -0,0 +1,36 @@
+// Can be done in mutple ways. Either read configuration from the command line arguments
+
+package config
```

Comment:
this gets rid of the yaml config feature. While I don't mind moving the code at all, let's just stick to the yaml config code for the template for now.

There is no reasonable way I can pass in the complete attendee-service configuration as command line arguments. It's complex and almost 400 lines long.

This code is not what we're going to need until maybe 8-12 months in the future, when the config service is ready, or when we're moving to k8s and have config maps.

So I'd like to keep the other code for now.

Reply:
The text is missleading, I actually had a bigger comment there, but for some reason half of it is missing.
By the time I thought we were using cli args and I was actually proposing to use either env files or yaml config files

Reply:
Also the config should only give an example of what kind of code you can have in the file. But it should be up to the developer, how to properly implement config structures per service

Reply:
- [ ] Update the comment in the file and indicate use of yaml config files